### PR TITLE
Add reference menu and linked to assets

### PR DESCRIPTION
**Pull request by AndaxH**

### CODE REVIEW

**Review by rissole — DISMISSED** [review]
let's CDN-ify instead of using images directly or we will hit free tier limits 😢

**Comment by bqx** [comment]
i like the use of the list but maybe we gotta rethink the UX of switching between lists (might be as simple as the list toggle button always sending you to the power/equip screen if its not active)

**Comment by rissole** [comment]
> i like the use of the list but maybe we gotta rethink the UX of switching between lists (might be as simple as the list toggle button always sending you to the power/equip screen if its not active)

this won't be an issue because the backpack button in the footer will switch the lists. the topbar "items" nav should just be the shop.

**Comment by AndaxH** [comment]
> let's CDN-ify instead of using images directly or we will hit free tier limits 😢

Updated to use links from google drive (https://drive.google.com/drive/folders/1GjEORs1H5pm6uYt2GWl6ubxoxQDmwDsl?usp=sharing) which seems to be working 🤞

**Comment by AndaxH** [comment]
> > let's CDN-ify instead of using images directly or we will hit free tier limits 😢
> 
> Updated to use links from google drive (https://drive.google.com/drive/folders/1GjEORs1H5pm6uYt2GWl6ubxoxQDmwDsl?usp=sharing) which seems to be working 🤞

Hmmmm yeah images don't load when I test in incognito - will investigate publicly accessible image links

**Comment by bqx** [comment]
> > i like the use of the list but maybe we gotta rethink the UX of switching between lists (might be as simple as the list toggle button always sending you to the power/equip screen if its not active)
> 
> this won't be an issue because the backpack button in the footer will switch the lists. the topbar "items" nav should just be the shop.

The issue is that the top nav "Reference" item takes you to a 3rd list now

File: src/components/ListReference/BossReference.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { FloatingBackButton, FullWidthListContainer } from ".";
+import FullWidthImageWithBackButton from "./FullWidthImageWithBackButton";
+
+// These are all ripped from our WoW Board Game Companion folder in Google Drive
+// no idea if these links are actually stable or not but ¯\_(ツ)_/¯
```
Comment (bqx) [inline src/components/ListReference/BossReference.tsx:6]: ¯\_(ツ)_/¯

File: src/components/ListReference/index.tsx [hunk 2]
```diff
@@ -1,11 +1,134 @@
...
+export const FloatingBackButton = styled.div`
+  position: absolute;
+  height: 64px;
+  width: 64px;
+  color: red;
+  font-size: 56px;
+  line-height: 56px;
+  font-weight: 900;
+  text-align: center;
+`;
```
Comment (bqx) [inline src/components/ListReference/index.tsx:81]: /nit: Can we make this consistent with the close button in the carousel? (X in the top right)

**Review by bqx — APPROVED** [review]

**Comment by AndaxH** [comment]
> > > i like the use of the list but maybe we gotta rethink the UX of switching between lists (might be as simple as the list toggle button always sending you to the power/equip screen if its not active)
> > 
> > 
> > this won't be an issue because the backpack button in the footer will switch the lists. the topbar "items" nav should just be the shop.
> 
> The issue is that the top nav "Reference" item takes you to a 3rd list now

yeah can rethink how this UI works but I think is okay as a v0

File: src/components/ListReference/index.tsx (hunk 2 above)
Comment (AndaxH) [reply src/components/ListReference/index.tsx:81]: yeah can make this consistent in a follow up PR

**Pull request merged**